.github/workflows/ci.yaml

```diff
@@ -21,6 +21,6 @@ jobs:
       - uses: open-turo/actions-gha/test@v2
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-      - uses: ./static-analysis
+      - uses: ./static-analysis/semgrep
```

Comment:
Rebase your branch on top of the latest `main`. The `/semgrep` folder doesn't exist anymore.